### DISCUSSION TITLE
用AI把關企業有無漂綠

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#326] [General]
若有ai把關，感覺會讓限制更嚴格，企業更能有效落實永續發展。
[https://ubrand.udn.com/ubrand/story/12117/5993348](https://ubrand.udn.com/ubrand/story/12117/5993348)

**Comment by cody910120** [comment]
我覺得會偷雞的還是會偷雞啦，雖然會更嚴格沒錯，單會做的還是會做，不會做得繼續找漏洞

**Comment by yt7l0** [comment]
有點好奇AI判斷企業「漂綠」的依據是什麼
